### PR TITLE
added language change

### PR DESCRIPTION
**Pull request by VolodymyrTrach**

### CODE REVIEW

File: src/pages/news/news-item/news-item.js [hunk 1]
```diff
@@ -11,31 +11,33 @@ import {
...
       : 'No author provided';
   const newsAuthorAvatar = author.image
     ? author.image.small
     : 'No author provided';
 
   const newsButtonText = ['читати далі', 'read more...'];
-  const newsDateLanguegeOptions = ['ukr-UA', 'en-US'];
-  const dateLanguege = `${newsDateLanguegeOptions[LANGUAGE]}`;
+  const newsDateLanguageOptions = ['ukr-UA', 'en-US'];
+  const dateLanguage = `${newsDateLanguageOptions[language]}`;
```
[the remark this answers is not in the document]
Comment (VolodymyrTrach) [reply src/pages/news/news-item/news-item.js:38]: You right. took template string ${...} out

File: src/pages/news/news-item/news-item.js [hunk 2]
```diff
@@ -11,31 +11,33 @@ import {
...
   const dateToShow = new Date(parseInt(date));
-  const newsDate = dateToShow.toLocaleString(`${dateLanguege}`, TIME_OPTIONS);
+  const newsDate = dateToShow.toLocaleString(`${dateLanguage}`, TIME_OPTIONS);
```
[the remark this answers is not in the document]
Comment (VolodymyrTrach) [reply src/pages/news/news-item/news-item.js:40]: You right. took template string ${...} out

[the remark this answers is not in the document]
Comment (VolodymyrTrach) [reply src/pages/news/news-item/news-item.js:40]: You right. took template string ${...} out

File: src/pages/news/news-detail/news-detail.js [hunk 3]
```diff
@@ -37,24 +39,24 @@ const NewsDetailPage = ({ match }) => {
...
+  const language = getFromLocalStorage('language');
   const newsTitle =
     article.title.length !== 0
-      ? article.title[LANGUAGE].value
+      ? article.title[language].value
       : 'No title provided';
-  const newsDateLanguegeOptions = ['ukr-UA', 'en-US'];
-  const dateLANGUAGE = `${newsDateLanguegeOptions[LANGUAGE]}`;
+  const newsDateLanguageOptions = ['ukr-UA', 'en-US'];
+  const dateLanguage = `${newsDateLanguageOptions[language]}`;
```
[the remark this answers is not in the document]
Comment (VolodymyrTrach) [reply src/pages/news/news-detail/news-detail.js:48]: You right. took template string ${...} out

File: src/pages/news/news-detail/news-detail.js [hunk 4]
```diff
@@ -37,24 +39,24 @@ const NewsDetailPage = ({ match }) => {
...
   const dateToShow = new Date(parseInt(article.date));
-  const newsDate = dateToShow.toLocaleString(`${dateLANGUAGE}`, TIME_OPTIONS);
+  const newsDate = dateToShow.toLocaleString(`${dateLanguage}`, TIME_OPTIONS);
```
[the remark this answers is not in the document]
Comment (VolodymyrTrach) [reply src/pages/news/news-detail/news-detail.js:50]: You right. took template string ${...} out

File: src/pages/news/news-detail/news-detail.js [hunk 5]
```diff
@@ -115,4 +118,14 @@ const NewsDetailPage = ({ match }) => {
   );
 };
 
+const newsDetailPropTypes = {
```
Comment (Alan18081) [inline src/pages/news/news-detail/news-detail.js:121]: There is no big reason to create a separate variables for props except cases where you are going to reuse it somewhere. You can just use `NewsDetailPage.propTypes = {...}`

File: src/pages/news/news-detail/news-detail.js [hunk 6]
```diff
@@ -115,4 +118,14 @@ const NewsDetailPage = ({ match }) => {
...
+  params: PropTypes.shape({
+    id: PropTypes.string
+  })
+};
+
+NewsDetailPage.propTypes = newsDetailPropTypes;
+
+NewsDetailPage.defaultProps = newsDetailPropTypes;
```
Comment (Alan18081) [inline src/pages/news/news-detail/news-detail.js:129]: `defaultProps` contains actual default values for the props

File: src/pages/news/news-detail/news-detail.js [hunk 7]
```diff
@@ -115,4 +118,14 @@ const NewsDetailPage = ({ match }) => {
   );
 };
 
+const newsDetailPropTypes = {
+  params: PropTypes.shape({
```
Comment (Alan18081) [inline src/pages/news/news-detail/news-detail.js:122]: The actual prop is `match` not the `params`. `params` exists inside the `match` object

File: src/pages/news/news-detail/news-detail.js (hunk 5 above)
Comment (VolodymyrTrach) [reply src/pages/news/news-detail/news-detail.js:121]: redone

File: src/pages/news/news-detail/news-detail.js (hunk 6 above)
Comment (VolodymyrTrach) [reply src/pages/news/news-detail/news-detail.js:129]: I made isRequired and we don't need default any more

File: src/pages/news/news-detail/news-detail.js (hunk 7 above)
Comment (VolodymyrTrach) [reply src/pages/news/news-detail/news-detail.js:122]: done

**Pull request merged**